includes/WC_Gateway_PayuInstallments.php

```diff
@@ -12,14 +12,81 @@ function __construct()
             if (!is_admin()) {
                 if (!$this->try_retrieve_banks()) {
                     add_filter('woocommerce_available_payment_gateways', [$this, 'unset_gateway']);
+                } else {
+                    wp_enqueue_style('payu-installments-widget', plugins_url( '/assets/css/payu-installments-widget.css', PAYU_PLUGIN_FILE ), [], PAYU_PLUGIN_VERSION);
+                    add_filter( 'woocommerce_gateway_title', [ $this, 'installments_filter_gateway_title' ], 10, 2);
                 }
             }
         }
     }
 
+    public function installments_filter_gateway_title( $title, $id )
+    {
+        wp_enqueue_script('payu-installments-widget', 'https://static.payu.com/res/v2/widget-mini-installments.js', ['jquery'], PAYU_PLUGIN_VERSION);
```

Comment:
Da radę przenieś to pod if-a razem z  wp_enqueue_style żeby nie ładowały niepotrzebne zasoby?

Reply:
done